lib/SetAPI/sampleset/SampleSetInterface.py

```diff
@@ -53,11 +53,6 @@ def create_sample_set(self, ctx, params):
 
         pprint(params)
 
-        conditionset_ref = params.get('conditionset_ref')
```

Comment:
removing condition_set from UI is fine. But can we keep this just in case?

Reply:
Maybe comment it out? I didn't want to leave too much unused stuff lying around that people might waste time trying to understand and get confused by.

Reply:
ConditionSet doesn't exist I think, and it sounds like it isn't really a priority

Reply:
👍  It only exists in CI.